runtime/aws/AWS Organizations/check-organization-ec2-tag-policies.ts

```diff
@@ -0,0 +1,82 @@
+import {
+	OrganizationsClient,
+	ListPoliciesCommand,
+	PolicyType
+} from "@aws-sdk/client-organizations";
+import { printSummary, generateSummary } from "../../utils/string-utils.js";
+import { ComplianceStatus, type ComplianceReport, type RuntimeTest } from "../../types.js";
+
+async function checkOrganizationTagPolicies(
+	region: string = "us-east-1"
+): Promise<ComplianceReport> {
+	const client = new OrganizationsClient({ region });
+	const results: ComplianceReport = {
+		checks: []
+	};
+
+	try {
+		const command = new ListPoliciesCommand({
```

Comment:
use helper from IAM folder

Reply:
The getAllIAMPolicies function is specifically designed for IAM policies (with Scope: "Local" and OnlyAttached: true), which is different from Organizations policies.